sql/moz-fx-data-shared-prod/udf/organic_vs_paid_desktop/udf.sql

```diff
@@ -0,0 +1,37 @@
+CREATE OR REPLACE FUNCTION udf.organic_vs_paid_desktop(medium STRING, adjust_network STRING)
+RETURNS STRING AS (
+  CASE
+    WHEN LOWER(medium) IN (
+        'paidsearch',
+        'paiddisplay',
+        'cpc',
+        'email',
+        'display',
+        'paidsocial',
+        'paidvideo',
+        'social',
+        'ppc',
+        'banner',
+        'ad-unit',
+        'newsletter'
```

Comment:
it seems this query is more about marketing attributable than paid... I suggest revisiting these categories.

Reply:
Sure - let me know which ones you want me to take out and can remove them